admin/test/package.scala

```diff
@@ -13,5 +14,7 @@ class AdminTestSuite
       new controllers.admin.DeploysControllerTest,
     )
     with SingleServerSuite {
-  override lazy val port: Int = 19015
+
+  override implicit lazy val portNumber: PortNumber = PortNumber(19015)
```

Comment:
We have to make this change because this test suite implements `ServerProvider` (via `SingleServerSuite`) and port is final in the new version of `scalatestplus-play` so it cannot be overriden: https://github.com/playframework/scalatestplus-play/blob/main/module/src/main/scala/org/scalatestplus/play/ServerProvider.scala#L48.